contrib/timm/test_timm.py

```diff
@@ -0,0 +1,19 @@
+import timm
+
+from ..utils import production_endpoint
+
+
+MODEL_ID = "nateraw/timm-resnet50-beans"
+
+
+def test_load_and_push_to_hub(repo_name: str, cleanup_repo: None) -> None:
```

Comment:
Should this be 2 tests?

Reply:
Done in https://github.com/huggingface/huggingface_hub/pull/1200/commits/173aff85bd26254a9a2d0d2cbc14f0d7c1f355f0.